README.md

```diff
@@ -21,19 +18,21 @@ Objdumped disassembly should be generated with lines info:
 
     objdump -d compiled.bin -l > disassembly.S
 
-Next, when you generated disassembly, you should tell to extension, where to
-search disassembly. By default it is searched right near the source file, but
-with `.S` extension:
+When you don't want to specify the output file manually (or use e.g. CMake or build scripts) and compile with GCC, you can use the [-save-temps option](https://gcc.gnu.org/onlinedocs/gcc/Developer-Options.html). This will put a `source.s` file into the output directory. Compile with:
+
+    $(CC) -g -save-temps $(CFLAGS) source.c
+
+Next, when you generated disassembly, you should tell the extension where to search for it. By default it is expected right next to the source file, but with `.S` extension:
 
 ```json
 "disasexpl.associations": {
     "**/*.c": "${fileDirname}/${fileBasenameNoExtension}.S",
     "**/*.cpp": "${fileDirname}/${fileBasenameNoExtension}.S"
 }
 ```
+Make sure to adjust the disassembly extension to `.s` when using the "-save-temps" option.
 
-Now, when all is done, simply select `F1 > Disassembly Explorer: Show`.
-Disassembly will be opened on the right side of your source.
+Now, when all is done, simply select `F1 > Disassembly Explorer: Show`. Disassembly will be opened on the right side of your source.
```

Comment:
Same as previous.
```suggestion
Now, when all is done, simply select `F1 > Disassembly Explorer: Show`.
Disassembly will be opened on the right side of your source.
```

Reply:
I wouldn't put so many unneeded line breaks here. If you watch the file on another device you want to make the text adapts to the screen as much as possible...

Reply:
Line breaks are present here to fit all the text in 80 symbols. That makes it possible to read it in _any_ editor, even without word wrap support (or with word wrap disabled).

On a device one probably will read rendered markdown instead of this raw text. And when it's rendered, this line breaks will not be visible.

Reply:
I personally don't see a use case for the 80 characters limit, but maybe that's just me. But anyway. Edit the pull request as needed. It's your project.